### PR TITLE
Updated CMS Documentation

### PR DESCRIPTION
**Pull request by ruslan-pan**

### CODE REVIEW

File: src/navigation/documentation.ts [hunk 1]
```diff
@@ -367,49 +367,55 @@ export const documentationNavigation: NavigationItem[] = [
...
-            url: '/docs/canvas/functions/cms-text',
-          },
-          {
-            title: 'cms_video',
-            url: '/docs/canvas/functions/cms-video',
+            title: 'Canopy',
+            items: [
+              {
+                title: 'cms',
+                url: '/docs/canvas/functions/canopy/cms',
```
Comment (jayan-blutui) [inline src/navigation/documentation.ts:374]: This is not a Canopy function.

Comment (dion-blutui) [reply src/navigation/documentation.ts]: Cool I will pull it out

File: src/pages/docs/canvas/functions/canopy/cms-audio.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+| `controls` (optional) | Display standard audio controls (play, pause, volume)                |   Boolean |
+| `autoplay` (optional) | Starts playback automatically when the page loads.                   |   Boolean |
+| `url` (optional)      | Link to the audio file                                               |    String |
+| `type` (optional)     | Specify the audio format (most browsers can handle multiple formats) |    String |
+| `class` (optional)    | A space-separated list of CSS classes to style the audio component   |    String |
+
+#### Multiple audio sources
+
+```canvas {% process=false %}
+{{ cms_audio(name, { controls: ..., autoplay: ..., sources: [{ url:'...', type:'...' },{ url: '...', type: '...' }], class: '...' }) }}
```
Comment (jayan-blutui) [inline src/pages/docs/canvas/functions/canopy/cms-audio.md:22]:
```suggestion
{{ cms_audio(name, { controls: ..., autoplay: ..., sources: [{ url: '...', type: '...' }, { url: '...', type: '...' }], class: '...' }) }}
```

File: src/pages/docs/canvas/functions/canopy/cms-button.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  Blutui AI
+</a>
+```
+
+#### Example
+
+{% code-group %}
+
+```canvas {% process=false filename="index.html" %}
+{{ cms_button('mybutton', { text: 'Blutui AI Components', url: 'https://blutui.ai/', class: 'bg-blue-500 text-white rounded hover:bg-blue-700',  opens_new_tab: true }) }}
```
Comment (jayan-blutui) [inline src/pages/docs/canvas/functions/canopy/cms-button.md:47]:
```suggestion
{{ cms_button('mybutton', { text: 'Blutui AI Components', url: 'https://blutui.ai/', class: 'bg-blue-500 text-white rounded hover:bg-blue-700', opens_new_tab: true }) }}
```

File: src/pages/docs/canvas/functions/canopy/cms-list.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  <li>Item 2</li>
+</ol>
+```
+
+#### Example
+
+{% code-group %}
+
+```canvas {% process=false filename="index.html" %}
+{{ cms_list('list-tag-123', { ordered: true, items:[ 'Item 1', 'Item 2', 'Item 3' ], class: 'list-decimal pl-5 space-y-2' }) }}
```
Comment (jayan-blutui) [inline src/pages/docs/canvas/functions/canopy/cms-list.md:45]:
```suggestion
{{ cms_list('list-tag-123', { ordered: true, items: [ 'Item 1', 'Item 2', 'Item 3' ], class: 'list-decimal pl-5 space-y-2' }) }}
```

File: src/pages/docs/canvas/functions/canopy/cms-video.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+| `controls` (optional) | Display standard video controls (play, pause, volume, etc.)          |   Boolean |
+| `autoplay` (optional) | Starts playback automatically when the page loads.                   |   Boolean |
+| `url` (optional)      | Link to the video file                                               |    String |
+| `type` (optional)     | Specify the video format (most browsers can handle multiple formats) |    String |
+| `class` (optional)    | A space-separated list of CSS classes to style the video component   |    String |
+
+#### Multiple video sources
+
+```canvas {% process=false %}
+{{ cms_video(name, { controls:..., autoplay: ..., sources: [{ url: '...', type: '...' },{url: '...', type: '...' }], class: '...' }) }}
```
Comment (jayan-blutui) [inline src/pages/docs/canvas/functions/canopy/cms-video.md:22]:
```suggestion
{{ cms_video(name, { controls: ..., autoplay: ..., sources: [{ url: '...', type: '...' },{ url: '...', type: '...' }], class: '...' }) }}
```

**Review by jayan-blutui — CHANGES_REQUESTED** [review]

**Review by jayan-blutui — APPROVED** [review]

**Pull request merged**